webanno/webanno
# #1370  Support curation from annotationpage in INCEpTION

Changes needed to integrate curation into annotation page in inception. Support for https://github.com/inception-project/inception/pull/1257

**What's in the PR**
* Extend VID for use in editor-extensions: VID format can either be previous format without optional extension-ID or contain extensionID, annotationID and extension payload. This payload is defined by the specific extension and parsed by it.
* Return result with addresses of updated/inserted annotationfs from casmerge
* Make sidebars optional
  * Re-factor annotationpage to init params before component creation -> can now check for curator status before configuring curation-sidebar
  * default implemented method in sidebars that check if sidebar should be added to page

* Closes https://github.com/webanno/webanno/issues/1370
* Closes https://github.com/webanno/webanno/issues/1401

**How to test manually**
* Test annotating, check that everything works as expected.
* Test curation, check that everything works as expected.

**Automatic testing**
* [x] PR updates unit tests
